distributed/shuffle/tests/test_graph.py

```diff
@@ -3,6 +3,11 @@
 import asyncio
 from typing import TYPE_CHECKING
 
+import pytest
+
+pd = pytest.importorskip("pandas")
+pytest.importorskip("dask.dataframe")
+
 import dask
 import dask.dataframe as dd
```

Comment:
```suggestion
import pytest

dd = pytest.importorskip("dask.dataframe")

import dask
```

Reply:
`mypy` is giving a 

```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

distributed/shuffle/tests/test_graph.py:25: error: Name "dd.DataFrame" is not defined
distributed/shuffle/tests/test_graph.py:26: error: Name "dd.DataFrame" is not defined
Found 2 errors in 1 file (checked 254 source files)
```

error. I'm going to revert to what I had originally, which `mypy` was happy with. Happy to do a follow-up to cleanup, but I'm confident in this PR and want to include it in the release today